knative-kubecon/README.md

```diff
@@ -230,4 +230,132 @@ $ curl -H "Host: helloworld-openshift.myproject.example.com" "http://$(minishift
 
 It's alive!
 
-**To be continued...**
\ No newline at end of file
+## 3. The Eventing Components
+
+The previous steps showed you how to invoke a Knative Serving application from an HTTP request, in that case submitted via curl.
+Knative Eventing is all about how you can invoke those functions in response to other events such as those received from message brokers or external applications.
+In this part of the demo we are going to show how you can receive Kubernetes platform events and route those to a Knative Serving application.
+
+Knative Eventing is built on top of three primitives:
+* Event Sources
+* Channels
+* Subscriptions
+
+_Event Sources_ are the components that receive the external events and forward them onto _Channels_.
+Out of the box we have _Channels_ that are backed by Apache Kafka, GCPPubSub and a simple in-memory channel.
+_Subscriptions_ are used to connect Knative Serving application to a _Channel_ so that it can respond to the events that the channel emits.
+
+Let's take a look at some of those resources in more detail.
+> Mention ServiceAccount?
+
+We have some yaml files prepared which describe the various resources, firstly the channel.
+
+```yaml
+$ less 010-channel.yaml
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Channel
+metadata:
+  name: testchannel
+spec:
+  provisioner:
+    apiVersion: eventing.knative.dev/v1alpha1
+    kind: ClusterChannelProvisioner
+    name: in-memory-channel
+```
+Here we can see that we've got a channel named `testchannel` and it is an `in-memory-channel` which is what we're going to use for this demo - in production we would probably use Apache Kafka.
+Let's deploy that so that we can use it in a later stage.
+
+```bash
+$ oc apply -f 010-channel.yaml -n myproject
```

Comment:
These should be fully qualified to contain the `eventing/` part in the path. Also the namespace argument is not necessary, since we're defaulting to `myproject`. We maybe should set that explicitly after the intro and evaling the oc env.